app/models.py

```diff
@@ -204,8 +236,14 @@ class DocumentEditState(db.Model):
     type = db.Column(db.String(), unique=True, nullable=False)
 
 
-def insert_default_values(types, model):
-    for t in types:
-        if db.session.query(model.id).filter_by(type=t).first() is None:
-            db.session.add(model(type=t))
-            db.session.commit()
+class DocumentEditModelSettings(db.Model):
+    __tablename__ = "DocumentEditModelSettings"
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    document_edit_id = db.Column(
+        db.Integer, db.ForeignKey("DocumentEdit.id"), nullable=False
+    )
+    model_id = db.Column(
```

Comment:
```suggestion
    recommendation_model_id = db.Column(
```

Man kann die Relationen in der DB deutlich leichter nachvollziehen, wenn sie eindeutig nach der relation benannt sind